src/app/Dashboard/CreatiIssue.tsx

```diff
@@ -0,0 +1,70 @@
+import { PageSection, Form, TextInput, FormGroup, Button, ActionGroup } from '@patternfly/react-core';
+import React, { useState } from 'react';
+import axios from 'axios'; 
+
+const CreateIssue: React.FunctionComponent = () => {
+    const [username, setUsername] = useState<string>("");
+
+    const [repositoryName, setRepositoryName] = useState<string>("");
+
+    const [title, setTitle] = useState<string>("");
+
+    const [body, setBody] = useState<string>("");
+
+    const [token, setToken] = useState<string>("");
+    
+    const onChange = (value) => {
+        setUsername(value);
+    }
+
+    const onChangeRepositoryname = (value) => {
```

Comment:
Add types to arguments as well
```suggestion
    const onChangeRepositoryname = (value: string) => {
```